AutofacSolution/Autofac.Entity/Class1.cs

```diff
@@ -6,7 +6,19 @@
 
 namespace Autofac.Entity
 {
-    public class Class1
+    public static class DbOperations
```

Comment:
Please use naming conventions. Class1.cs should also be renamed to DbOperations.cs